docs/conf.py

```diff
@@ -206,3 +210,25 @@
     # This page is under login
     r"https://readthedocs\.org/accounts/gold",
 ]
+
+
+# Config values for sphinxcontrib-mermaid are handled differently
+# If running on a local environment, we render "raw" and bundle mermaid-js from CDN to render diagrams in the browser.
+# This is to avoid requiring the @mermaid-js/mermaid-cli package
+#
+# This can run locally but the following issues need to be solved:
+# a) Read the Docs Docker images need at least libasound2 to be installed for puppeteer to work.
+# b) A <style> with FontAwesome needs to be injected into the SVG's DOM.
+if False and os.environ.get("READTHEDOCS"):
+
+    # Write SVGs, those we can style via CSS! (we aren't doing that yet)
+    mermaid_output_format = "svg"
+
+    # Do not load javascript from CDN
+    mermaid_version = ""
+
+    # Nothing to init, we don't build raw diagrams
+    mermaid_init_js = ""
+
+# Use transparent backgrounds
+mermaid_params = ["--backgroundColor", "transparent"]
```

Comment:
I'll remove all of this at the end of this PR. It seems we cannot benefit from the extension, unless we resolve how to run mermaid-cli in RTD docker containers.

Reply:
What about using Sphinx Timeline to express the steps required for the DNS: https://sphinx-timeline.readthedocs.io/en/latest/ ?